### PR TITLE
darwin: fix memory leak in uv_cpu_info

### PR DESCRIPTION
**Pull request by yjhjstz**
see #536

### CODE REVIEW

File: src/unix/darwin.c [hunk 1]
```diff
@@ -199,8 +199,10 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   }
 
   *cpu_infos = uv__malloc(numcpus * sizeof(**cpu_infos));
-  if (!(*cpu_infos))
+  if (!(*cpu_infos)) {
+    vm_deallocate(mach_task_self(), (vm_address_t)info, msg_type);
     return -ENOMEM;  /* FIXME(bnoordhuis) Deallocate info? */
+  }
```
Comment (saghul) [inline src/unix/darwin.c]: You can delete the comment now.

**Comment by saghul** [comment]
LGTM with a nit

**Comment by yjhjstz** [comment]
update.

**Comment by saghul** [comment]
:+1:

**Comment by bnoordhuis** [comment]
LGTM.  I wonder why I only put in a comment instead of fixing it right away.

**Comment by saghul** [comment]
Landed in 9c307cc, thanks a lot!

**Pull request closed without merging**